david-davis.txt

```diff
@@ -0,0 +1,106 @@
+
+https://codepen.io/DCdavis/pen/qBEyrGQ?editors=1100
+
+    1. What is Semantic HTML? 
+    HTML is the structure on the page.
+
+    2. What is HTML used for?
+    HTML is used to build the page.
+
+    3. What is an attribute and where do we put it? 
+    attributes are used in opening tags to control an elements behavior.
+
+    4. What is the h1 tag used for? How many times should I use it on a page?
+    The h1 tag is used for the most important heading on the page and they should only be used once.
+
+    5. Name two tags that have required attributes
+      nav and img both use attributes.
+
+    6. What do we put in the head of our HTML document? 
+    Title, links to other pages, things that you dont want displayed on the page, meta tages, links to font.
+    7. What is an id? 
+    8. What elements can I add an id to? 
+    You can add id's to any element.
+    9. How many times can I use the same id on a page? 
+    You can only use an id once.
+
+    10. What is a class? 
+    A class is used to put on elements for styling purposes.
+
+    11. What elements can I add a class to? 
+    Class can be used on any html element.
+
+    12. How many times can I use the same class on a page?
+    You can use a class as many times as you like.
+
+    13. How do I get my link to open in a new tab?
+    Use the src attribute to connect to a link that goes to a different page.
```

Comment:
If you only have the src attribute, the link will only on the same tab that you have open.  Can you think of another attribute you can add to make it open in a new tab?